projects/xerces-c/Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM gcr.io/oss-fuzz-base/base-builder
+MAINTAINER vincent.ulitzsch@live.de
+RUN apt-get update && apt-get install -y make autoconf automake libtool wget zlib1g-dev
+RUN apt-get update && \
+apt-get install -y  libtool ninja-build cmake subversion
+RUN apt-get install -y libstdc++6
```

Comment:
Just curious, do you know why libstdc++6 is required at all? I'm surprised it is. Does LPM need it? Maybe if LPM were built with `-stdlib=libc++` it wouldn't need it?